cardano-cli/test/Test/Golden/Shelley/MultiSig/Address.hs

```diff
@@ -20,8 +20,8 @@ golden_shelleyAllMultiSigAddressBuild = propertyOnce . H.moduleWorkspace "tmp" $
   allMultiSigFp <- noteInputFile "test/data/golden/shelley/multisig/scripts/all"
 
   allMultiSigAddress <- execCardanoCLI
-    [ "address", "build-script"
-    , "--script-file", allMultiSigFp
+    [ "address", "build"
+    , "--payment-script-file", allMultiSigFp
```

Comment:
Should a message about this change go in the changelog?

Reply:
There is one: https://github.com/input-output-hk/cardano-node/pull/4700/files#diff-4ed4667555ef45301f33907270e1ff1a1038d53987fea0f0cc4e585d170043a8R4